Gopkg.toml

```diff
@@ -1,3 +1,7 @@
+[[constraint]]
+  branch = "TOOL-2241-move-testresultexport-to-separate-lib"
```

Comment:
should be changed to master before merge but this branch needs to be merged first